force-app/main/default/layouts/ServiceSession__c-Service Session Layout.layout-meta.xml

```diff
@@ -2,6 +2,7 @@
 <Layout xmlns="http://soap.sforce.com/2006/04/metadata">
     <excludeButtons>ChangeOwnerOne</excludeButtons>
     <excludeButtons>ChangeRecordType</excludeButtons>
+    <excludeButtons>PrintableView</excludeButtons>
```

Comment:
Was this change intentional or leftover from a previous iteration?

Reply:
This was intentional since we are not using the standard PrintableView button and we are using the custom one.

Reply:
But then it looks like we have it added to the layout down below in the sorting? It is still showing on the layout.

Reply:
🤦‍♀️  I forgot to commit those changes. I am just seeing it now.